examples/nrf52840/build.rs

```diff
@@ -10,14 +10,77 @@
 
 use std::env;
 use std::fs::File;
-use std::io::Write;
+use std::io::{BufWriter, Write};
 use std::path::PathBuf;
 
+fn hex_to_bytes(s: &str) -> Option<Vec<u8>> {
```

Comment:
hex::encode?

Reply:
Yeah, I initially didn't want to include extra dependency for just a single semi-trivial function (`left-pad` :dagger: )..